src/components/utils/browserUtils.js

```diff
@@ -0,0 +1,7 @@
+// checks whether user is on mobile browser
+function isMobile() {
```

Comment:
I think it might be better to use something like https://www.npmjs.com/package/react-responsive for responsive designs. Also since we only really care about the size of the screen.  I'm going to use this package too for timeline

Here are some common breakpoints: https://stackoverflow.com/questions/16443380/common-css-media-queries-break-points

but, i think it might also be good to try finding the one that works best for groups in the coup specifically by gradually decreasing the width of your screen and seeing when the desktop design breaks.